code-challenges.js

```diff
@@ -12,16 +12,38 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
-
-
+// create a function that takes a number 
+const threeMe = (num) => {
+    if(num % 3 === 0) {
+// see if this number is divisible by three or not (if statment)
+        return `${ num } is divisible by three`
+    } else {
+// if it is not return something that lets them know
+        return `${ num } is NOT divisible by three`
+    }
+}
+console.log(threeMe(num1))
+console.log(threeMe(num2))
+console.log(threeMe(num3))
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
-
+const capNouns = (arr) => {
+    //create a variable to split array
+    //create a new empty array for the new array to be pushed into
+    var nounsNowCap = []
+//iterate through that array using and capitalize the first letter in every word
+    for(let i = 0; i < arr.length; i++) {
+        nounsNowCap.push(randomNouns[i].charAt(0).toUpperCase()+ randomNouns[i].slice(1))
+    }
+    //return the newly capitalized words into a new array because you don't want to modify original data.
+    return nounsNowCap
+}
+console.log(capNouns(randomNouns))
```

Comment:
Wonderful! I'd look into seeing how you can refactor this by using a higher-order function! (Hint: Map!)